spec/views/posts/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "posts/edit", type: :view do
+  before(:each) do
+    @post = assign(:post, Post.create!(
+      title: "MyString",
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/FirstArgumentIndentation: Indent the first argument one step more than Post.create!(.